WcaOnRails/app/views/regulations/history/index.html.erb

```diff
@@ -5,22 +5,23 @@
-    <li><%= link_to "2014-04-03", "./official/2014-04-03/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2014-01-01...official-2014-04-03#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/regulations-and-guidelines-april-3-2014" %>)
+    <li><%= link_to "2016-04-18", "https://www.worldcubeassociation.org/regulations/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-regulations/compare/official-2015-07-01...official-2016-04-18#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-april-2016" %>)</li>
+    <li><%= link_to "2015-07-01", "./official/2015-07-01/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-regulations/compare/official-2014-04-03...official-2015-07-01#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-july-2015" %>)</li>
+    <li><%= link_to "2014-04-03", "./files/2014-04-03/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2014-01-01...official-2014-04-03#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/regulations-and-guidelines-april-3-2014" %>)
       <ul>
-        <li><%= link_to "2014-01-01", "./official/2014-01-01/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/diff-base-2013...official-2014-01-01#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-and-guidelines-2014" %>)</li>
+        <li><%= link_to "2014-01-01", "./files/2014-01-01/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/diff-base-2013...official-2014-01-01#files_bucket" %>, <%= link_to "Summarized Changes", "https://www.worldcubeassociation.org/posts/wca-regulations-and-guidelines-2014" %>)</li>
       </ul></li>
-      <li><%= link_to "2013-05-21", "./official/2013-05-21/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2013-05-10...official-2013-05-21#files_bucket" %>)
+      <li><%= link_to "2013-05-21", "./files/2013-05-21/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2013-05-10...official-2013-05-21#files_bucket" %>)
         <ul>
-          <li><%= link_to "2013-05-10", "./official/2013-05-10/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2013-01-01...official-2013-05-10#files_bucket" %>)</li>
-          <li><%= link_to "2013-01-01", "./official/2013-01-01/" %></li>
+          <li><%= link_to "2013-05-10", "./files/2013-05-10/" %> (<%= link_to "Changes", "https://github.com/cubing/wca-documents/compare/official-2013-01-01...official-2013-05-10#files_bucket" %>)</li>
+          <li><%= link_to "2013-01-01", "./files/2013-01-01/" %></li>
```

Comment:
Why have you changed `official` to `files`? I can see the directories (*dates*) [here](https://github.com/cubing/worldcubeassociation.org/tree/regulations-data/regulations/build/regulations/history/official).